pmt/include/pmt/pmtf.hpp

```diff
@@ -23,6 +23,7 @@ class pmt_base : public std::enable_shared_from_this<pmt_base>
 
     bool serialize(std::streambuf& sb)
     {
+        serialize_setup();
```

Comment:
There is a `bm_pmt_serialize_uvec.cpp` to do some limited measurements of serialization performance.  It will be good to keep our eye on the impact this has - not clear to me whether it will have any.